hack/upgrade-test.sh

```diff
@@ -41,10 +41,19 @@
 # to verify that it is updated to the new operator image from 
 # the local registry.
 
-MAX_STEPS=8
+MAX_STEPS=9
 CUR_STEP=1
 RELEASE_DELTA="${RELEASE_DELTA:-1}"
 HCO_DEPLOYMENT_NAME=hco-operator
+HCO_NAMESPACE="kubevirt-hyperconverged"
```

Comment:
missing 
`HCO_CATALOG_NAMESPACE=openshift-marketplace`

?

Reply:
it's in hack/upgrade-openshiftci-config